src/pages/Login/Login.jsx

```diff
@@ -37,38 +71,48 @@ function Login () {
     }
   ]
   return (
-        <div className="login">
-            <Jumbotron
-                pageTitle={'Login'}
-                backgroundClassName="login__bg"
-                breadcrumb={breadcrumb}
-            />
-            <div className="login-container">
-                <h2 className="login__title">Login</h2>
-                <div className="login__form">
-                    <form onSubmit={handleLogin}>
-                        <label htmlFor="username" className="login__label">Username or email</label>
-                        <input id='username' type="text" className="login__input" placeholder='Username or Email' value={username} onChange={handleUsernameChange} required/>
+    <div className="login">
+      <Jumbotron
+        pageTitle={'Login'}
+        backgroundClassName="login__bg"
+        breadcrumb={breadcrumb}
+      />
+      <div className="login-container">
+        {
+          loading &&
+          <img
+            src='https://res.cloudinary.com/dzmkilinu/image/upload/v1692693479/medical-site/loading_xcwewp.gif'
+            style={{ width: '200px', height: '200px', textAlign: 'center' }}
+          />
+        }
+        {
+          !loading && <>
```

Comment:
In JSX, it's better for conditional rendering surround the component with parenthesis.